components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/form-builder/forms/stream-form.js

```diff
@@ -110,6 +39,345 @@ define(['require', 'log', 'jquery', 'lodash', 'attribute', 'stream', 'designView
             }
         };
 
+        const REGEX = /^([a-zA-Z])$/;
```

Comment:
provide a varibaleName which describes this Regex eg:- whitespaceValidatorRegex

Reply:
done